programs/marginfi/src/state/price.rs

```diff
@@ -1021,4 +1184,92 @@ mod tests {
                 .unwrap()
         );
     }
```

Comment:
Lets add some tests that test the oracle price parsing and all cases with or without wide price bands

Reply:
You mean prices with PriceBias::Low/High? Have that included at the end of test `swb_pull_get_price`.